resources/home/dnanexus/utils/util_functions.py

```
@@ -1,22 +1,124 @@
 """General utility functions"""
 
+import os
+from typing import Tuple, Union
```

Ruff (0.8.0)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)